src/ukify/ukify.py

```diff
@@ -47,7 +47,7 @@
                     IO,
                     Optional,
                     Sequence,
-                    Union)
+                    Union, List, Tuple)
```

Comment:
These imports shouldn't be necessary. `ukify` already requires Python >= 3.9 and there you can use `list` and `tuple` for type annotations instead